formats/specialist_document/frontend/examples/utaac-decision.json

```diff
@@ -0,0 +1,53 @@
+{
+  "content_id": "3917cb0e-0928-45b7-bf63-20f9bc113a7b",
+  "base_path": "/administrative-appeals-tribunal-decisions/example-utaac-decision",
+  "title": "Example UTAAC Decision",
+  "description": "Example text.",
+  "format": "specialist_document",
+  "need_ids": [],
+  "locale": "en",
+  "updated_at": "2016-04-05T14:41:08.625Z",
+  "public_updated_at": "2016-04-05T14:41:08.000+00:00",
+  "details": {
+    "metadata": {
```

Comment:
I took this JSON from a locally running content-store instance, e.g. http://content-store.dev.gov.uk/content/administrative-appeals-tribunal-decisions/example-utaac-decision . My understanding is that this is the correct approach to get a realistic example file.

You're right, there are no values for `tribunal_decision_categories_name`, `tribunal_decision_judges_name`, and `tribunal_decision_sub_categories_name`. Not sure why that is.

Though in my locally running finder frontend, those values are populated correctly:
![screen shot 2016-04-12 at 15 35 58](https://cloud.githubusercontent.com/assets/3447/14463775/4e0e6afa-00c4-11e6-813e-26ef1b5b3f33.png)

Did you want to check on the integration content store to see what the JSON output is? I'm not sure of the url for the integration content-store.